### PR TITLE
Fix some scalability thresholds

### PR DESCRIPTION
**Pull request by wojtek-t**
/assign @shyamjvs

### CODE REVIEW

File: sig-scalability/configs-and-limits/thresholds.md [hunk 1]
```diff
@@ -45,12 +45,12 @@ Important notes about the numbers:
 | Number of pods per node<sup>1</sup> | 110            |             | 500            |
 | Number of pods per core<sup>1</sup> | 10             |             | 10             |
 | Number of namespaces (ns)           | 10000          |             | 100000         |
-| Number of pods per ns               | 15000          |             | 50000          |
+| Number of pods per ns               | 5000           |             | 50000          |
```
[the remark this answers is not in the document]
Comment (wojtek-t) [reply sig-scalability/configs-and-limits/thresholds.md]: 10k are from daemonsets, 5k are static pods (which are a bit different).

Also - that's the namespace where nothing really happens after we do, so I'm not convinced we should be announcing that widely.

[the remark this answers is not in the document]
Comment (shyamjvs) [reply sig-scalability/configs-and-limits/thresholds.md]: Yeah. Let's neglect those.
Btw - we had decided upon 3k limit for this (in the slides).

[the remark this answers is not in the document]
Comment (wojtek-t) [reply sig-scalability/configs-and-limits/thresholds.md]: OK - changed to 3k.

**Comment by jeremyeder** [comment]
@wojtek-t @shyamjvs What is the justification for lowering these limits?

**Comment by sjug** [comment]
@shyamjvs What test are you referring to specifically getting slower? Having a single RC scaling pod count up?

**Comment by shyamjvs** [comment]
@sjug Controllers have 2 way dependencies across pods and RCs (or whatever equivalent of RC):

- every pod change triggers a "label-selector match" operation against each RC being maintained
- every RC change triggers a "list all pods in namespace" call

So having a higher number of either pods or RCs per namespace will slow down the controller. We don't have exact quantitative values, but an empirical observation from our load test (where we're creating 100s of RCs and 1000s of pods per ns).

**Pull request merged**